packages/framework/request-handler/src/runtimeRequestHandlerBuilder.ts

```diff
@@ -41,7 +41,7 @@ export class RuntimeRequestHandlerBuilder {
 }
 
 /**
- * @deprecated Will be removed in future major release. Migrate all usage of IFluidRouter to the "entryPoint" pattern. Refer to Removing-IFluidRouter.md
+ * @deprecated Will be removed once Loader LTS version is "2.0.0-internal.7.0.0". Migrate all usage of IFluidRouter to the "entryPoint" pattern. Refer to Removing-IFluidRouter.md
```

Comment:
Can we switch it to `@internal`?  Then once we get our exports split up it'll be essentially removed in terms of the public API surface.  cc @Josmithr for any commentary on that process / timeline.